components/raftstore/src/store/peer_storage.rs

```diff
@@ -154,6 +153,7 @@ impl EntryCache {
         if entries.is_empty() {
             return;
         }
+        let current_mem_size = self.get_mem_size();
```

Comment:
Should monitor changes instead of calculate it every time.

Reply:
I am confusing about monitor changes. Could you please give some advice about monitor changes?

Reply:
For example, only calculate the newly appended entries and removed entries.

Reply:
Now I think it's only calculate mem size change when the capacity changed.